tracing-core/Cargo.toml

```
@@ -36,6 +36,7 @@ maintenance = { status = "actively-developed" }
 
 [dependencies]
 once_cell = { version = "1.13.0", optional = true }
+valuable = { path = "../../valuable/valuable" }
```

Just have this for testing locally with my copy of Valuable, which is ahead of master

---

instead of using a local path dep (which makes it hard for anyone else to test this change), you could replace the dependency with a Git dep on your branch:
```suggestion
valuable = { git = "https://github.com/bryangarza/valuable", branch = "partialeq-namedfield" }
```
that way, we can run the tests on CI, and so on.

it would probably be preferable to leave the dependency in the `Cargo.toml` unchanged, and use a workspace-level [cargo `[patch]`](https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html) for this.

---

Thanks for the comment! I'll commit the suggestion and see if I can add a cargo patch on the next commit